### ISSUE TITLE
R 3.2.3 on Mac OS 10.11.3 - Error: IO error: error writing bytes to file

### ISSUE THREAD

**Issue by kmakat** [wesm/feather#100]
Hi,

Writing a 140,000,000 x 16 data.table to disk quits with:

`Error: IO error: error writing bytes to file` after 8 GB have been written to disk.

`read_feather` on this file returns  in `Error: Invalid: Feather file footer incomplete`.

Thanks,
Kemal

``` r
path = "dt.feather"
R> write_feather(dt_all_reads, path)
Error: IO error: error writing bytes to file
```

``` r
R version 3.2.3 Patched (2016-02-09 r70141)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: OS X 10.11.3 (El Capitan)

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] feather_0.0.0.9000 colorout_1.1-1     setwidth_1.0-4    

loaded via a namespace (and not attached):
[1] assertthat_0.1 tools_3.2.3    tibble_1.0     Rcpp_0.12.4
```

**Comment by hadley** [comment]
Can you please include a reproducible example, by generating random data and avoiding the use of data table (to eliminate that as a possible cause)

**Comment by kmakat** [comment]
I tried, but generally it seems to work, with or without `data.table`. Performance is great and
ideal for my case. Not sure what did not work, I tried it repeatedly (within the same R session though).
Will re-try later today.

```r
R> set.seed(123)
R> my_df = data.frame(replicate(16, sample(0:100000, 150000000, rep = TRUE)))
R> library(feather)
R> write_feather(my_df, "~/Desktop/my_df.feather") # works
R> library(data.table)
data.table 1.9.6  For help type ?data.table or https://github.com/Rdatatable/data.table/wiki
The fastest way to learn (by data.table authors): https://www.datacamp.com/courses/data-analysis-the-data-table-way
R> my_dt = data.table(my_df) # works also
R> write_feather(my_dt, "~/Desktop/my_dt.feather")
```

P.S. Reading both feather files into new R session also worked!

**Comment by wesm** [comment]
Any updates here?

**Comment by wesm** [comment]
This should be fixed in Feather V2 (coming in `arrow` 0.17.0)
